components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/EndpointCertificatesApiService.java

```diff
@@ -1,31 +1,15 @@
 package org.wso2.carbon.apimgt.rest.api.publisher.v1;
 
-import org.wso2.carbon.apimgt.rest.api.publisher.v1.*;
-import org.wso2.carbon.apimgt.rest.api.publisher.v1.dto.*;
-
```

Comment:
are these imports get removed during code generation or are we removing the unused imports manually?

Reply:
Code generation